README.md

```diff
@@ -57,6 +57,8 @@ credentials:
 
 ### Releasing a new version
 
+> Note: You can exclude merge PRs in the change log by using `[MERGE]` inside the title of the PR & Commit.
```

Comment:
This functionality will only be part of a specific implementation so it does not belong in this readme.

Reply:
I'll remove it then

Reply:
@VSlokker Can you explain why this should not be documented? This is generic behavior that is applied for all projects this tool is used on. That should be in the README otherwise people have to browse the code to figure out why their merge requests that have `[MERGE]` in the title are missing in the changelog.

Also it is an optional feature that you can decide to use when using the release-tool, how would you know about this feature without browsing the code?

Reply:
As this feature is only available based on the type of changelog generator being used (which i know; currently is always the PR changelog generator). It is dependent on that specific implementation of the changelog generator and not an actual feature of the release tool itself.

Maybe we can document it somewhere, but it should be very clear that this is tied to the PR changelog generator implementation.